en/08_Changelogs/alpha/6.0.0-alpha1.md

```diff
@@ -1131,6 +1134,8 @@ If you were calling `getSchemaData()` in your `getAttributes()` method in a `For
 
 ### Full list of removed and changed API (by module, alphabetically) {#api-removed-and-changed}
 
+<!-- markdownlint-disable proper-names enhanced-proper-names -->
```

Comment:
This is the same treatment the list of commits gets. I'll update cow later to do this automatically as part of https://github.com/silverstripe/.github/issues/345